server.js

```diff
@@ -60,6 +75,23 @@ function Weather(forecastObj) {
   this.time = forecastObj.valid_date;
 }
 
+function Trail(trailObj) {
+  this.name = trailObj.name;
+  this.location = trailObj.location;
+  this.length = trailObj.length;
+  this.stars = trailObj.stars;
+  this.star_votes = trailObj.starVotes;
+  this.summary = trailObj.summary;
+  this.trail_url = trailObj.url;
+  this.conditions = `${trailObj.conditionStatus}: ${trailObj.conditionDetails}`;
```

Comment:
Can you help me understand the use of the `$ notation for your this.conditions value?